docs/reference-guide/query-modules/implement-custom-query-modules/api/python-api.md

```diff
@@ -180,457 +180,1596 @@ RETURN example.func_example("first argument");
 
 Naturally, you may pass in different arguments.
 
-## class mgp.Properties(vertex_or_edge)
-Bases: `object`
+This module provides the API for usage in custom openCypher procedures.
 
-A collection of properties either on a Vertex or an Edge.
+## Label Objects
 
+```python
+class Label()
+```
+
+Label of a `Vertex`.
+
+#### name
```

Comment:
## class mgp.read_proc(func: Callable[[…], mgp.Record]) -> i would make these titles only class mgp.read_proc()

i would make #### into a ### so they are visible in the right-side menu

Reply:
maybe add brackets to functions? just an idea